dataactvalidator/config/sqlrules/b9_award_financial.sql

```diff
@@ -1,18 +1,14 @@
 WITH award_financial_b9_{0} AS
 	(SELECT submission_id,
 		row_number,
-		beginning_period_of_availa,
```

Comment:
Yes it should be removed already, I forgot to take remove that from select statement. So I clean this up with this hotfix.